REVIEW.md

```diff
@@ -0,0 +1,61 @@
+# Guidance for Reviewing Pull Requests in Notary Project
+
+This document covers who may review pull requests for this project, and provides guidance on how to perform reviews that meet our community standards and code of conduct. All reviewers must read this document and agree to follow the project review guidelines. Reviewers who do not follow these guidelines may have their privileges revoked.
+
+## The Reviewer Role
+
+Everyone is encouraged to review code and documentation changes, even if they are not officially designated as reviewers. However, only pull requests approved by maintainers can be merged.
+
+## Process
+
+1. Do not start reviewing a pull request if it is WIP or is a draft mode until it is marked as **ready to review**.
+2. Do not start reviewing a pull request if automated checks fail. Wait until they are fixed by the author.
+3. Pull requests except those generated by bots are suggested openning an issue associated. Motivations and designs SHOULD be discussed in the corresponding issue instead of in the pull request. Minor changes may be exempt at the maintainers' discretion.
+4. Do a quick check for the license header for all new files.
+5. When you provide feedback, make it clear if the change must be made for the pull request to be approved, or if it is just a suggestion. 
+6. Reviews from CODEOWNERS in each repo are required. Only pull requests approved by **at least one repo maintainer** can be merged. If the author is a maintainer, the pull request is required to be approved by other maintainers.
+7. Maintainers can merge their own pull requests after being approved by other maintainers and meet the requirement of branch protection rules.
+8. Pull requests are required to be reviewed again if updates come after the approval from maintainers.
+
+## Checklist
+
+Below are a set of common questions that apply to all pull requests:
+
+- [ ] Is this PR targeting the correct branch?
+- [ ] Does the commit message provide an adequate description of the change?
+- [ ] Does the affected code have corresponding tests?
+- [ ] Are the changes well documented and recognizable by `godoc`?
```

Comment:
This assumes all code in the repository is written in Go. Is this what are we targeting?